mabwiser/mab.py

```diff
@@ -667,6 +667,25 @@ def _is_compatible(self, learning_policy: LearningPolicy):
                                                 LearningPolicy.ThompsonSampling))
 
 
+LearningPolicyType = NewType('LearningPolicyType', Union[LearningPolicy.EpsilonGreedy,
```

Comment:
Mind adding a brief comment like "LearningPolicyType is the union of all learning policies"